generator.sh

```diff
@@ -7,8 +7,9 @@ if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
     exit 1
 fi
 
-script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-less $script_dir/LICENSE
+curl -s -O -L https://raw.githubusercontent.com/mobify/generator-astro/develop/LICENSE
+less LICENSE
+rm LICENSE
```

Comment:
Instead of just doing `rm LICENSE` here, let's put this right after we `curl` the LICENSE:
```
trap 'rm -f LICENSE' EXIT
```
That will guarantee that the file is deleted no matter how the script exits.